src/WebJobs.Script/functionTemplate.js

```diff
@@ -1,9 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-var f = require('{0}');
+var f = require('{0}'),
+    util = require('util');
 
 return function (context, callback) {{
+    var origLog = context.log;
+    context.log = function(value) {{
+        origLog(util.inspect(value));
```

Comment:
Fixed and pushed to master. Code is now:

```javascript
var origLog = context.log;
context.log = function(value) {{
    if (!util.isString(value)) {{
        value = util.inspect(value);
    }}
    origLog(value);
}};
```

Feel free to sync and give it a try locally. Let me know if you run into any more issues.